src/main/docker/Dockerfile.jre

```diff
@@ -21,13 +21,17 @@ FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-base:2
 # Refresh the OS repositories and install OpenJDK 17 Runtime Environment
 RUN zypper -n refresh && \
     zypper -n update && \
-    zypper -n install java-17-openjdk && \
+    zypper -n install java-17-openjdk mozilla-nss-tools && \
     zypper -n clean --all
 
 # Install Java certificate installation script
 ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.19.0/container-cert-script/install-ca-cert-java.sh \
     /startup/startup.d/
 RUN chmod +x /startup/startup.d/install-ca-cert-java.sh
 
+# Add the startup scripts
+COPY startup.d/ /startup/startup.d/
+RUN chmod +x /startup/startup.d/*
```

Comment:
Don't make this a startup script as we don't want it to happen when the services start.  It should already be done and included in the base image.